cantools/database/can/formats/dbc.py

```diff
@@ -381,6 +383,18 @@ def attributes(self):
     def attributes(self, value):
         self._attributes = value
 
+    @property
+    def comment(self):
+        """The DBC specific comment as a string.
+
+        """
+
+        return self._comment
+
+    @comment.setter
+    def comment(self, value):
+        self._comment = value
+
```

Comment:
What's the reason for adding the comment attribute to the DbcSpecifics class? Isn't it better to use the comment attribute on the Bus class?